src/bindings/channel_header.ts

```diff
@@ -35,12 +35,12 @@ const channelHeaderSubscribe = (): AppBinding => {
     } as AppBinding;
 };
 
-const channelHeaderConfig = (): AppBinding => {
+const channelHeaderConfig = (context: AppContext): AppBinding => {
     return {
         location: CommandLocations.Configure,
         label: 'Configure Zendesk',
         description: 'Open Create Zendesk Config Modal',
-        icon: ZDIcon,
+        icon: getStaticURL(context, 'zendesk.png'),
```

Comment:
Why not keep ZDIcon as `zendesk.png`? (That way if we change back to svg we only have to change it in one place)

Reply:
Good suggestion :+1: